lighthouse-core/test/results/sample_v2.json

```diff
@@ -2232,7 +2232,6 @@
             "node": {
               "type": "node",
               "selector": "h2",
-              "path": "3,HTML,1,BODY,5,DIV,0,H2",
```

Comment:
this seems unexpected. I think you'll need to update the artifacts with the new name in https://github.com/GoogleChrome/lighthouse/blob/a58510583acd2f796557175ac949932618af49e7/lighthouse-core/test/results/artifacts/artifacts.json#L1315


(maybe hold off until https://github.com/GoogleChrome/lighthouse/pull/11405/files#r487451543 is settled though?)